Sources/ApolloCodegenLib/Glob.swift

```diff
@@ -0,0 +1,196 @@
+import Foundation
+
+private extension String {
```

Comment:
worth making this an extension on `ApolloExtensions` even though it's private so that it's clear this is functionality we added?

Reply:
If it's private, I don't think we need to do that. I'd definitely agree if it was shared outside of this file.

Reply:
I'm fine with that